swarm/storage/mru/handler.go

```diff
@@ -204,7 +177,7 @@ func (h *Handler) New(ctx context.Context, request *Request) error {
 	request.rootAddr = chunk.Addr
 
 	h.chunkStore.Put(ctx, chunk)
-	log.Debug("new resource", "name", request.metadata.Name, "startTime", request.metadata.StartTime, "frequency", request.metadata.Frequency, "owner", request.metadata.Owner)
+	log.Debug("new resource", "name", request.metadata.Topic, "startTime", request.metadata.StartTime, "frequency", request.metadata.Frequency, "owner", request.metadata.Owner)
```

Comment:
All this code is part of the `New()` function, is commented out and going away. I didn't remove it yet to avoid confusion during the review, but I guess it did create the confusion, so deleting it now.